vnext/local-cli/runWindows/utils/commandWithProgress.js

```diff
@@ -41,6 +41,27 @@ function newSpinner(text) {
   return ora(options).start();
 }
 
+async function runPowerShellScriptFunction(text, script, funcName, verbose) {
+  try {
+    await commandWithProgress(
+      newSpinner(text),
+      text,
+      'powershell',
+      [
+        '-NoProfile',
+        '-ExecutionPolicy',
+        'RemoteSigned',
+        `Import-Module "${script}"; ${funcName} -ErrorAction Stop; exit $LASTEXITCODE`,
+      ],
+      verbose,
+    );
+  } catch {
+    // The error output from the process will be shown if verbose is set.
+    // We don't capture the process output if verbose is set, but at least we have the task name in text, so throw that.
+    throw new Error(text);
```

Comment:
This exception message seems potentially confusing. What about something like ``` `${text} command failed with error ${ex.message}` ```

Reply:
We don't get an exception message - we only get back that the task failed (since this is based on process exit code).

The "command failed" part is prepended/appended by the consumer of this helper since they have more context

---
In reply to: [405939498](https://github.com/microsoft/react-native-windows/pull/4536#discussion_r405939498) [](ancestors = 405939498)